src/FacultyDirectory.Core/Services/ScholarService.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using AngleSharp.Html.Dom;
+using AngleSharp.Html.Parser;
+using FacultyDirectory.Core.Data;
+
+namespace FacultyDirectory.Core.Services
+{
+    public interface IScholarService
+    {
+        Task<SourceData> GetTagsAndPublicationsById(string id);
+    }
+
+    public class ScholarService : IScholarService
+    {
+        private readonly ApplicationDbContext dbContext;
+        private readonly HttpClient httpClient;
+
+        public ScholarService(ApplicationDbContext dbContext, HttpClient httpClient)
+        {
+            this.dbContext = dbContext;
+            this.httpClient = httpClient;
+        }
+
+        public async Task<SourceData> GetTagsAndPublicationsById(string id)
+        {
+            if (string.IsNullOrWhiteSpace(id)) {
+                throw new ArgumentNullException();
+            }
+
+            var siteUrl = $"https://scholar.google.com/citations?hl=en&user={id}";
```

Comment:
@srkirkland App setting? :)

Reply:
@jSylvestre whole lot of app settings will be needed :)